js/views/userPageVw.js

```diff
@@ -1546,22 +1542,7 @@ module.exports = baseVw.extend({
   },
 
   close: function(){
```

Comment:
I think it would be safer to make this method called `remove` and then if we want to offer a `close` method, we could add in a close method that calls remove, e.g:

```
close: function() {
  this.remove();
}
```

That way, in case someone removes this view (by calling remove()), the remove functionality (colorbox close, etc...) is still called. The way it is now, they would always have to call close.